pkg/Elmah.AspNet.nuspec

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<package xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <metadata xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+        <id>elmah.aspnet</id>
+        <version>2.0.0-alpha001</version>
+        <title>ELMAH AspNet Library</title>
+        <authors>Atif Aziz</authors>
+        <owners>Atif Aziz, James Driscoll</owners>
+        <licenseUrl>http://www.apache.org/licenses/LICENSE-2.0</licenseUrl>
+        <projectUrl>https://elmah.github.io/</projectUrl>
+        <requireLicenseAcceptance>false</requireLicenseAcceptance>
+        <description>ELMAH (Error Logging Modules and Handlers) AspNet.</description>
+        <summary>ELMAH (Error Logging Modules and Handlers) AspNet without Elmah.CoreLibrary.</summary>
```

Comment:
This can be removed. IIRC, the [`<summary>` tag is being deprecated](https://github.com/NuGet/Home/issues/4587#issuecomment-288913324) and the description is short and sweet enough.